script.js

```diff
@@ -0,0 +1,77 @@
+const form = document.querySelector('#form');
+const ul = document.querySelector('#book-list');
+
+// Get data from localstorage
+const parsedData = JSON.parse(localStorage.getItem(('books')));
+const books = parsedData;
+
+// Add book to browser
+function addBooktoArray(item, index) {
+  const li = document.createElement('li');
+  li.classList.add('list-item');
+  li.setAttribute('key', index);
+  li.innerHTML = `
+  <h3 class="title">${item.title}</h3>
+  <p class="author">${item.author}</p>
+  <button class="removeBtn">Remove</button><hr>`;
+
+  ul.appendChild(li);
+}
+
+// Display books on website
+function displayBooks() {
+  books.forEach((item, index) => {
+    addBooktoArray(item, index);
+  });
+}
+
+// remove book
+function removeBook(element) {
+  if (element.classList.contains('removeBtn')) {
+    element.parentElement.remove();
+    const bookID = element.parentElement.getAttribute('key');
+    const result = books.filter((book) => book !== books[bookID]);
```

Comment:
It has a function that removes a book from the collection 🔥 Awesome use of the `filter()` method!